db/seeds.rb

```diff
@@ -24,3 +24,9 @@
 Question.create(body: question3, answer: answer3, quiz: quiz2)
 Question.create(body: question4, answer: answer4, quiz: quiz2)
 Question.create(body: question5, answer: answer5, quiz: quiz2)
+
+Team.create(name: 'team1', quiz: quiz1)
+Team.create(name: 'team2', quiz: quiz1)
+Team.create(name: 'team3', quiz: quiz1)
+Team.create(name: 'team4', quiz: quiz1)
+Team.create(name: 'team5', quiz: quiz1)
```

Comment:
So that Viktoria could see more than one team in the list, to make sure we're listing it right.. Want me to remove it?